pkg/tanka/workflow.go

```diff
@@ -90,6 +90,41 @@ func Diff(baseDir string, mods ...Modifier) (*string, error) {
 	return kube.Diff(l.Resources, opts.diff)
 }
 
+// Delete parses the environment at the given directory (a `baseDir`) and deletes
+// the generated objects from the Kubernetes cluster defined in the environment's
+// `spec.json`.
+func Delete(baseDir string, mods ...Modifier) error {
+	opts := parseModifiers(mods)
+
+	l, err := load(baseDir, opts)
+	if err != nil {
+		return err
+	}
+	kube, err := l.connect()
+	if err != nil {
+		return err
+	}
+	defer kube.Close()
+
+	// show diff
+	// static differ will never fail and always return something if input is not nil
+	diff, err := kubernetes.StaticDiffer(false)(l.Resources)
```

Comment:
`err` is unused here.

I suggest still checking it, as it will be fatal if it occurs, as this most probably means `diff(1)` is missing from the system

Reply:
Fixed.